specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-11-01/virtualMachineScaleSet.json

```diff
@@ -5963,6 +5970,16 @@
           "$ref": "./computeRPCommon.json#/definitions/HardwareProfile",
           "description": "Specifies the hardware settings for the virtual machine."
         },
+        "resilientVMDeletionStatus":{
```

Comment:
enum should be model-as-string: true. 
please find an example of that in the file and follow it